bigquery_etl/glean_usage/templates/baseline_clients_first_seen.view.sql

```diff
@@ -0,0 +1,9 @@
+{{ header }}
+
+CREATE OR REPLACE VIEW
+  `{{ first_seen_view }}`
```

Comment:
I don't think we should create this user-facing view right now. Once we get the first seen tables created and backfilled, we'll need to update the daily and last_seen queries to pull in `first_seen_date` and materialize a boolean `is_first_seen` field for clustering. The `baseline_clients_first_seen` view should point at `baseline_clients_daily_v1` and filter for `is_first_seen` being set.

Reply:
The script logic at the moment does best when there is a user facing view; it's a little tricky to decouple since the view is dry-run to get the table reference. I don't see too much harm in exposing this if it's going to be a subset of the final set of columns.